tools/cndump/kvset_dump.h

```diff
@@ -0,0 +1,17 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+/*
+ * Copyright (C) 2022 Micron Technology, Inc.  All rights reserved.
+ */
+
+#ifndef CNDUMP_KVSET_DUMP_H
+#define CNDUMP_KVSET_DUMP_H
+
+struct hse_kvdb;
+
+unsigned int
+kvset_cmd_parse_args(int argc, char **argv);
+
+void
+kvset_cmd(struct hse_kvdb *kvdb);
```

Comment:
Why do the commands not parse the args? This doesn't make sense to expose the function

Reply:
Good point.  That's an artifact of how it morphed into the current shape.

Reply:
good suggestion.  fixed.